app/controllers/entities_controller.rb

```diff
@@ -23,6 +27,19 @@ def create
     end
   end
 
+  def update
+    respond_to do |format|
+    @entity = Entity.find(params[:id])
```

Comment:
Wrong indentation

Reply:
To eliminate duplication, move this code into `def set_entity`

`@entity = Entity.find(params[:id])`

Add before action on top.